database/src/main/kotlin/no/nav/su/se/bakover/database/grunnlag/GrunnlagPostgresRepo.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.su.se.bakover.database.grunnlag
+
+internal class GrunnlagPostgresRepo(
+    private val uføregrunnlagRepo: UføregrunnlagRepo
+) : GrunnlagRepo,
+    UføregrunnlagRepo by uføregrunnlagRepo
```

Comment:
Vil det ikke bli kræsj med delegation her etter hvert dersom andre repo også har metoder som `hent`, `lagre` osv?

Reply:
Eller, `lagre` vil kanskje gå bra siden de da vil overloades på hva som skal lagres. `hent(behandlingId: UUID)` blir vel fortsatt kræsj da.

Reply:
Ikke helt sikker på hvorfor det endte opp slik, antar jeg hadde en plan om å unngå å måtte injecte så mange repos for tester ellerno. Vil nok kunne få konflikter ja, men det bør vel være ganske greit å håndtere (eksplisitte navn på funksjoner og/eller referere til spesifikke repo).